pom.xml

```diff
@@ -148,8 +148,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.12.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
Let's leave that at 1.7 for now...

Reply:
@saudet I wish I could, but maven on my machines refuses to build 1.7 source/target.